### PR TITLE
addPermissionsToRole function

### PR DESCRIPTION
**Pull request by djamesfar**
added a function addPermissionsToRole which acccepts array of permission strings and role string, assigns/creates permissions to passed role string.

### CODE REVIEW

File: src/Permissions/BasePermissionsMigration.php [hunk 1]
```diff
@@ -25,4 +25,26 @@ public function createPermissions($permissions)
...
+
+            app(PermissionRegistrar::class)->forgetCachedPermissions();
+
+            foreach ($permissions as $permission) {
+                /** @psalm-suppress UndefinedMethod */
+                app(Permission::class)::findOrCreate($permission, null);
+                $Role->givePermissionTo($permission);
+            }
+        }
+    }
```
Comment (drewroberts) [inline src/Permissions/BasePermissionsMigration.php:49]: This looks great!

**Review by drewroberts — APPROVED** [review]
Thank you! 🐠

File: src/Permissions/BasePermissionsMigration.php (hunk 1 above)
Comment (djamesfar) [reply src/Permissions/BasePermissionsMigration.php:49]: :+1:

**Comment by drewroberts** [comment]
Issue #23

**Pull request merged**